javascript/packages/orchestrator/src/cmdGenerator.ts

```diff
@@ -279,37 +275,95 @@ export async function genCmd(
   if (bootnodes && bootnodes.length)
     args.push("--bootnodes", bootnodes.join(" "));
 
-  // port flags logic
-  const portFlags = {
-    "--prometheus-port": nodeSetup.prometheusPort,
-    "--rpc-port": nodeSetup.rpcPort,
-    "--ws-port": nodeSetup.wsPort,
-  };
-
-  for (const [k, v] of Object.entries(portFlags)) {
-    args.push(...[k, v.toString()]);
-  }
-  args.push(...["--listen-addr", `/ip4/0.0.0.0/tcp/${nodeSetup.p2pPort}/ws`]);
-
   // set our base path
   const basePathFlagIndex = args.findIndex((arg) => arg === "--base-path");
   if (basePathFlagIndex >= 0) args.splice(basePathFlagIndex, 2);
   args.push(...["--base-path", dataPath]);
 
-  const finalArgs: string[] = [
-    command,
-    "--chain",
-    `${cfgPath}/${chain}.json`,
-    "--name",
-    name,
-    "--rpc-cors",
-    "all",
-    "--unsafe-rpc-external",
-    "--rpc-methods",
-    "unsafe",
-    "--unsafe-ws-external",
-    ...args,
-  ];
+  let portFlags
+  let finalArgs: string[]
+  if (command.includes("gossamer")) {
```

Comment:
I feel like this is all unnecessary if we just support the substrate flags.  The `command` can be modified to run our binary easily.